src/shogun/converter/ica/FastICA.cpp

```diff
@@ -112,11 +113,11 @@ void CFastICA::fit_dense(CDenseFeatures<float64_t>* features)
 	if (m_mixing_matrix.num_rows != m || m_mixing_matrix.num_cols != m)
 	{
 		m_mixing_matrix = SGMatrix<float64_t>(m,m);
-
+		NormalDistribution<float64_t> normal_dist;
```

Comment:
will there be an array filling routine for normal rng? That's quite useful to have
@vigsterkr @theartful I didn't follow the discussions

Reply:
@karlnapf it can be implemented later.

Reply:
but the interface we try to keep as std

Reply:
@karnlapf yeah, I already added it, just need to push :")